### PR TITLE
PS-8782 5.7.42-46 Release notes and release note tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	new file:   docs/release-notes/5.7.42-46.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: docs/release-notes/5.7.42-46.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Improvements and bug fixes provided by Oracle for MySQL 5.7.42 and included in Percona Server for MySQL are the following:
+
+* In InnoDB, online DDL operations are prevented from accessing out-of-bounds memory
+  
+Find the full list of bug fixes and changes in the [MySQL 5.7.42 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-42.html).
+
+## Bug fixes
+
+* [PS-8779](https://jira.percona.com/browse/PS-8779): The `mysqldump` client utility ignored `--set-gtid-purged=OFF`.
```
Comment (percona-ysorokin) [inline docs/release-notes/5.7.42-46.md]: Wrong Jira ticket here
the correct one is https://jira.percona.com/browse/PS-8776

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: docs/release-notes/5.7.42-46.md (hunk 1 above)
Comment (patrickbirch) [reply docs/release-notes/5.7.42-46.md]: Fixed

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**